packages/website/src/components/BigNumbers/BigNumbers.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { VStack } from '@coderscamp/ui/components/Stack';
+
+import { StatsWithTitle } from './StatsWithTitle';
+
+const amounts = {
+  currentEdition: ['7.', '1516', '200', '6'],
+  previousEditions: ['6526', '1022', '372', '502'],
+};
+
+const labels = {
+  currentEdition: ['edycja', 'zadań i materiałów', 'miejsc na kurs', 'projektów do stworzenia'],
+  previousEditions: ['zgłoszeń', 'osób na kursie', 'wydanych certyfikatów', 'zakodowanych projektów'],
+};
```

Comment:
💭 I think it could be more readable if you define statistics as two arrays of objects (e.g. `currentEdition` and `previousEditions`). Each object in the array could use `StatisticProps` interface. Doing it this way gives you the following benefits:
1. In `StatsWithTitle.tsx` you can simply map the array instead of creating four Statistic elements manually.
2. StatsWithTitle can receive only two props - `title` and `statistics`.
3. You can destructure objects when passing them to the Statistic component instead of defining each prop separately.